### PR TITLE
AVS and Voter-Dapp read same variable for determining transaction limits on batch commits and reveals

### LINKED ISSUE #846: Build a common utility that all dApps and core scripts use to read batch transaction limits
The AVS and the Voting dApp should use the same maximum number of commits & reveals that they can fit into one batch. 

Currently, they are each defining their own hard-coded limits

Related to closed PR's #827 and #826

### PR DESCRIPTION
**Pull request by nicholaspai**
Fix #846

### CODE REVIEW

File: voter-dapp/src/ActiveRequests.js [hunk 1]
```diff
@@ -251,22 +252,25 @@ function ActiveRequests({ votingAccount, votingGateway }) {
...
       totalSelected += checkboxesChecked[checked];
     }
-    // This number defines the maximum number of transactions that can be fit within one block.
-    // It was calculated by testing the batchCommit and batch revealFunctions to identify the
-    // maximum that can be placed within one tx. The actual number that can be fit is slightly
-    // more but is set to a lower amount for safety here.
-    return totalSelected <= 25;
+    return totalSelected <= BATCH_MAX_COMMITS;
+  };
+  const canExecuteBatchReveal = () => {
```
Comment (mrice32) [inline voter-dapp/src/ActiveRequests.js]: nit: just to make this a little more DRY, wdyt about just making one function and pass the limit in as an argument?

File: voter-dapp/src/ActiveRequests.js [hunk 2]
```diff
@@ -367,7 +371,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
       ) : (
         ""
       )}
-      {!canExecuteBatch() ? (
+      {(saveButtonShown && !canExecuteBatchCommit()) || (revealButtonShown && !canExecuteBatchReveal()) ? (
         <span style={{ paddingLeft: "10px", color: "#FF4F4D" }}>
           You can only commit or reveal up to 25 requests at once. Please select fewer.
```
Comment (ptare) [inline voter-dapp/src/ActiveRequests.js]: Parameterize the 25 in this text as well.

File: voter-dapp/src/ActiveRequests.js (hunk 1 above)
Comment (nicholaspai) [reply voter-dapp/src/ActiveRequests.js]: done

File: voter-dapp/src/ActiveRequests.js (hunk 2 above)
Comment (nicholaspai) [reply voter-dapp/src/ActiveRequests.js]: done

**Comment by nicholaspai** [comment]
@mrice32 Please see my change to `Constants.js` regarding `MAX_SAFE_JS_INT`

**Pull request merged**